### PR TITLE
Enforce tagify whitelist by default

### PR DESCRIPTION
**Pull request by CarlosFdez**

### CODE REVIEW

File: src/util/tags.ts [hunk 1]
```diff
@@ -42,6 +42,7 @@ function tagify(
         return null;
     }
 
+    enforceWhitelist ??= true;
```
Comment (stwlam) [inline src/util/tags.ts:45]: Why not up top?

Comment (CarlosFdez) [reply src/util/tags.ts:45]: No particular reason, just seemed pretty. I can move it.

File: src/util/tags.ts [hunk 2]
```diff
@@ -36,6 +36,8 @@ function tagify(
     input: HTMLInputElement | null,
     { whitelist, maxTags, enforceWhitelist }: TagifyOptions = {}
 ): Tagify<TagRecord> | null {
+    enforceWhitelist ??= true;
```
Comment (stwlam) [inline src/util/tags.ts:39]: hiiigher

Comment (CarlosFdez) [reply src/util/tags.ts]: Funny enough I independently noticed what you meant. I think I wasn't aware you could set defaults for objects destructured in the params.

**Pull request merged**